app/models/data_model/publication.rb

```diff
@@ -0,0 +1,7 @@
+module DataModel
+	class Publication < ActiveRecord::Base
+		include Genome::Extensions::UUIDPrimaryKey
+		has_and_belongs_to_many :interactions
```

Comment:
I think you are right. In that case you will want to add here all of the other many-to-many relationships, e.g. to genes etc.